File: app/src/main/java/com/samsung/android/ecgmonitor/EcgAverager.java
package com.samsung.android.ecgmonitor;


public final class EcgAverager {
    private double lastAvg = 0;

    public double update(double newAvg) {
        lastAvg = newAvg;
        return lastAvg;
    }

    public double get() {
        return lastAvg;
    }
}

File: app/src/main/java/com/samsung/android/ecgmonitor/base/model/EcgAverager.kt
package com.samsung.android.ecgmonitor.base.model


class EcgAverager {
    private var lastAvg = 0.0

    fun update(newAvg: Double): Double {
        lastAvg = newAvg
        return lastAvg
    }

    fun get(): Double {
        return lastAvg
    }
}
